src/tests/access/test_ownable_twostep.cairo

```diff
@@ -318,19 +339,18 @@ fn test_pending_accept_after_owner_renounce() {
 // Helpers
 //
 
-fn assert_event_ownership_transfer_started(
-    previous_owner: ContractAddress, new_owner: ContractAddress
-) {
-    let event = utils::pop_log::<OwnableComponent::Event>(ZERO()).unwrap();
-    let expected = OwnableComponent::Event::OwnershipTransferStarted(
-        OwnershipTransferStarted { previous_owner, new_owner }
-    );
-    assert!(event == expected);
-    utils::assert_no_events_left(ZERO());
-
-    let mut indexed_keys = array![];
-    indexed_keys.append_serde(selector!("OwnershipTransferStarted"));
-    indexed_keys.append_serde(previous_owner);
-    indexed_keys.append_serde(new_owner);
-    utils::assert_indexed_keys(event, indexed_keys.span());
+#[generate_trait]
+impl TwoStepSpyHelpersImpl of TwoStepSpyHelpers {
+    fn assert_event_ownership_transfer_started(
+        ref self: EventSpy,
+        from_address: ContractAddress,
+        previous_owner: ContractAddress,
+        new_owner: ContractAddress
+    ) {
+        let expected = OwnableComponent::Event::OwnershipTransferStarted(
+            OwnershipTransferStarted { previous_owner, new_owner }
+        );
+        self.assert_emitted_single(from_address, expected);
+    // TODO(immrsd): Add keys check later
```

Comment:
I don't think it looks good or is worth adding to many TODO comments, and this specific one would need to be added to every event assertion for consistency. I vote we remove it (we have another like this in another file).

Reply:
Removed